Building RustPython requires Rust 1.36

RustPython currently makes use of `VecDeque rotate_left`, and `VecDeque rotate_right`, which were made stable in Rust 1.36.0. Attempting to build RustPython with Rust 1.35.0 results in a build error.

```
error[E0658]: use of unstable library feature 'vecdeque_rotate' (see issue #56686)
   --> vm/src/stdlib/collections.rs:193:19
    |
193 |             deque.rotate_left(-mid as usize);
    |                   ^^^^^^^^^^^

error[E0658]: use of unstable library feature 'vecdeque_rotate' (see issue #56686)
   --> vm/src/stdlib/collections.rs:195:19
    |
195 |             deque.rotate_right(mid as usize);
    |                   ^^^^^^^^^^^^

error: aborting due to 2 previous errors
```

I'm not suggesting this is a bug that requires fixing, I just wanted to document it for others. The problem goes away with `rustup update`. Feel don't hesitate to close this issue without further action.

<details>

```
:~/src/RustPython master± cargo run demo.py
  Downloaded env_logger v0.5.13
  Downloaded indexmap v1.0.2
...
   Compiling rustpython-compiler v0.1.0 (/home/alex/src/RustPython/compiler)
   Compiling rustpython-derive v0.1.0 (/home/alex/src/RustPython/derive)
error[E0658]: use of unstable library feature 'vecdeque_rotate' (see issue #56686)
   --> vm/src/stdlib/collections.rs:193:19
    |
193 |             deque.rotate_left(-mid as usize);
    |                   ^^^^^^^^^^^

error[E0658]: use of unstable library feature 'vecdeque_rotate' (see issue #56686)
   --> vm/src/stdlib/collections.rs:195:19
    |
195 |             deque.rotate_right(mid as usize);
    |                   ^^^^^^^^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0658`.
error: Could not compile `rustpython-vm`.

To learn more, run the command again with --verbose.
:~/src/RustPython master 1± rustup show
Default host: x86_64-unknown-linux-gnu

installed toolchains
--------------------

stable-x86_64-unknown-linux-gnu
nightly-x86_64-unknown-linux-gnu

active toolchain
----------------

stable-x86_64-unknown-linux-gnu (default)
rustc 1.35.0 (3c235d560 2019-05-20)

:~/src/RustPython master± rustup update
info: syncing channel updates for 'stable-x86_64-unknown-linux-gnu'
info: latest update on 2019-07-04, rust version 1.36.0 (a53f9df32 2019-07-03)
...
info: checking for self-updates

   stable-x86_64-unknown-linux-gnu updated - rustc 1.36.0 (a53f9df32 2019-07-03)
  nightly-x86_64-unknown-linux-gnu updated - rustc 1.38.0-nightly (4560cb830 2019-07-28)

:~/src/RustPython master± cargo run demo.py
   Compiling libc v0.2.60
   Compiling autocfg v0.1.5
 ...
   Compiling rustpython-compiler v0.1.0 (/home/alex/src/RustPython/compiler)
   Compiling rustpython-derive v0.1.0 (/home/alex/src/RustPython/derive)
   Compiling rustpython v0.1.0 (/home/alex/src/RustPython)
     Finished dev [unoptimized + debuginfo] target(s) in 2m 17s
     Running `target/debug/rustpython demo.py`
Hello, RustPython!
:~/src/RustPython master± git show --abbrev-commit | cat
commit c8ce3bde
Merge: 50dd93db 42dca443
Author: Noah <33094578+coolreader18@users.noreply.github.com>
Date:   Mon Jul 29 08:05:48 2019 -0500

    Merge pull request #1187 from RustPython/dict-keying
    
    Implement dictionary indexing by trait.
```

</details>

--- a/README.md
+++ b/README.md
@@ -16,6 +16,10 @@ A Python-3 (CPython >= 3.5.0) Interpreter written in Rust :snake: :scream:
 
 #### Check out our [online demo](https://rustpython.github.io/demo/) running on WebAssembly.
 
+RustPython requires Rust 1.36 or higher. 
+To check Rust version: `rustc --version` If you wish to update,
+`rustup update stable`.
+
 To test RustPython, do the following:
 
     $ git clone https://github.com/RustPython/RustPython

